src/main/java/org/springframework/data/couchbase/core/convert/CouchbaseCustomConversions.java

```diff
@@ -112,9 +110,6 @@ public static CouchbaseCustomConversions create(Consumer<CouchbaseConverterConfi
 
 	@Override
 	public boolean hasValueConverter(PersistentProperty<?> property) {
-		if (property.findAnnotation(Encrypted.class) != null) {
-			return true;
-		}
 		return super.hasValueConverter(property);
```

Comment:
super.hasValueConverter() eventually calls CouchbasePropertyValueConverterFactory.getConverter() which does this check.